src/lib389/lib389/utils.py

```diff
@@ -583,7 +600,14 @@ def valgrind_disable(sbin_dir):
                          e.strerror)
 
     # Enable selinux
-    os.system('setenforce 1')
+    if os.geteuid() == 0:
+        os.system('setenforce 1')
+
+    # Restore .inf file (for systemd)
+    new_path = Paths()._get_defaults_loc(DEFAULTS_PATH)
+    old_path = f'{new_path}.orig'
+    if os.exists(old_path):
```

Comment:
Indeed, I just wonder why the test that I run on 1 minutetips was PASS instead of getting an AttributeError exception
 (probably because it was the finalizer that was impacted .. )